bin/compile

```diff
@@ -42,8 +42,8 @@ install_promscale() {
 
   echo -n "Checking Promscale executable checksum..."
   curl --silent --fail --location --retry 3 --retry-delay 2 --connect-timeout 3 --max-time 30 --output ${CACHE_DIR}/${promscale_executable}.sha256 "${promscale_url}/${promscale_checksums}"
-  if [[ $(sha256sum ${CACHE_DIR}/promscale | cut -f 1 -d " ") != $(cat ${CACHE_DIR}/${promscale_executable}.sha256 | grep $promscale_executable | cut -f 1 -d " ") ]]; then
-    echo 2>&1 "Wrong checksum"
+  if [[ $(sha256sum ${CACHE_DIR}/promscale | cut -f 1 -d " ") != $(cat ${CACHE_DIR}/${promscale_executable}.sha256 | grep "$promscale_executable$" | cut -f 1 -d " ") ]]; then
```

Comment:
Out of curiosity, what the purpose of the `$"` at the end of the `grep`?
If the goal was to find the character `$` then I would have put a `\` in front.

Reply:
It's a regexp, and `$` means end of line. Hence we are looking for the text in `$promscale_executable` at the end of a line.